Sources/SkeletonView.swift

```diff
@@ -211,13 +224,32 @@ extension UIView {
     }
     
     func removeSkeletonLayer() {
+        guard isSkeletonActive,
+            let skeletonLayer = skeletonLayer else { return }
+        guard let transitionType = currentSkeletonConfig?.transition else {
+            removeSkeletonLayerFinalize()
+            return
+        }
+        
+        //Handle transition
+        switch transitionType {
```

Comment:
if we become public `startTransition...` method, we could use here, then we wouldn't have a lot of switch blocks